src/io/internal/pipelines/dicom/CMakeLists.txt

```diff
@@ -10,6 +10,27 @@ find_package(ITK REQUIRED
   )
 include(${ITK_USE_FILE})
 
+set(cpp_nlohmann_json_GIT_REPOSITORY "https://github.com/nlohmann/json.git")
+set(cpp_nlohmann_json_GIT_TAG  bc889afb4c5bf1c0d8ee29ef35eaaf4c8bef8a5d)
+FetchContent_Declare(
+  cpp_nlohmann_json
+  GIT_REPOSITORY ${cpp_nlohmann_json_GIT_REPOSITORY}
+  GIT_TAG        ${cpp_nlohmann_json_GIT_TAG}
+)
+
+FetchContent_MakeAvailable(cpp_nlohmann_json)
+
+#find_package(cpp-base64 REQUIRED)
+set(cpp_base64_GIT_REPOSITORY "https://github.com/thewtex/cpp-base64.git")
```

Comment:
This is currently a duplicate. `compress-stringify` also uses cpp-base64. So we'll need to move this higher up the dir structure and use it as a common target for both.